tox.ini

```diff
@@ -1,11 +1,55 @@
 [tox]
-skipsdist = True
+envlist=code-check
```

Comment:
WE also had a `pytest-check`, we want to add it later?

Reply:
yes, we will add it, after we add some tests. This work would be ongoing and iterative.